gbpservice/nfp/modules/configurator.py

```diff
@@ -261,6 +261,9 @@ def get_notifications(self, context):
         """
 
         notifications = self.sc.get_notification()
+        if notifications:
```

Comment:
Lets us dump notifications even if it is empty. Also, let us make it a debug log.